src/components/RegisterLanding.jsx

```diff
@@ -0,0 +1,80 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import NoOnboarding from '../pages/NoOnboarding';
+import StyledFieldset from './StyledFieldset';
+
+export default function RegisterLanding({ setRegisterStep }) {
+  const [registered, registerSet] = useState(null);
+
+  const handleRadioChoice = (e) => {
+    if (e.target.id === 'radio1') {
+      registerSet(true);
+    } else if (e.target.id === 'radio2') {
+      registerSet(false);
+    }
+  };
+
+  return (
+    <>
+      <StyledFieldset>
+        <h1 id="registerh1">Volunteer with us at Code for Chicago!</h1>
+
+        <div id="registerd1">
+          <p>
+            As part of the Code for America brigade network, Code for Chicago connects skills-based
+            volunteers to Chicago-based nonprofits, mutual aid groups, and people with ideas on how
+            they can improve their community.
+          </p>
+          <p id="registerp2">
+            This contact form is for people who have completed our Onboarding Night and are
+            interested in participating in Code for Chicago projects and committees. We created this
+            formal process to cut down on the onboarding time. Our hope is to assign you to a
+            project or committee within a week of your submission. If you don&apos;t fit a project
+            need or none of the projects interest you then we will put you on a waiting list and
+            reach out when new projects come up.
+          </p>
+        </div>
+
+        <hr />
+
+        <h2 id="registerh2">Have you registered and attended Onboarding Night?</h2>
+        <label className="statusLabel" htmlFor="radio1">
+          <input
+            type="radio"
+            name="status"
+            id="radio1"
+            value="yes"
+            onChange={(e) => handleRadioChoice(e)}
+          />
+          Yes
+        </label>
+        <label className="statusLabel" htmlFor="radio2">
+          <input
+            type="radio"
+            name="status"
+            id="radio2"
+            value="no"
+            onChange={(e) => handleRadioChoice(e)}
+          />
+          No
+        </label>
+        <nav>
+          <button
+            id="registerb1"
+            type="button"
+            onClick={registered ? () => setRegisterStep(2) : () => registerSet('not')}
+            disabled={registered === null}
+          >
+            Next
+          </button>
+        </nav>
+
+        {registered === 'not' && <NoOnboarding />}
```

Comment:
< NoOnboarding /> needs to replace the content instead of being appended to it:

<img width="815" alt="Screen Shot 2021-08-12 at 3 32 24 PM" src="https://user-images.githubusercontent.com/9326234/129265602-a1b673b9-158c-4ee9-845b-2bf1a418b96b.png">